src/common/procedure/src/local/runner.rs

```diff
@@ -942,7 +943,14 @@ mod tests {
 
         // Run the runner and execute the procedure.
         runner.run().await;
-        let err = meta.state().error().unwrap().to_string();
+        let err = meta
```

Comment:
NIT: unwrap safe?

Reply:
Yes. Even if the error doesn't contain any source field, `last().unwrap()` would still return last error found, which is itself.